Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -766,11 +860,14 @@ CGMutablePathRef CGPathCreateMutableCopyByTransformingPath(CGPathRef path, const
 
     if (transform && !CGAffineTransformEqualToTransform(*transform, CGAffineTransformIdentity)) {
         CGMutablePathRef transformedPath = CGPathCreateMutable();
-        path->ClosePath();
+        FAIL_FAST_IF_FAILED(path->ClosePath());
+
+        transformedPath->SetStartingPoint(CGPointApplyAffineTransform(path->GetStartingPoint(), *transform));
 
+        transformedPath->BeginFigure();
```

Comment:
You mean without us closing it first?

---
In reply to: [96993664](https://github.com/Microsoft/WinObjC/pull/1738#discussion_r96993664) [](ancestors = 96993664)